include/openssl/x509.h.in

```diff
@@ -404,12 +404,12 @@ int X509_NAME_digest(const X509_NAME *data, const EVP_MD *type,
 
 X509 *X509_load_http(const char *url, BIO *bio, BIO *rbio, int timeout);
 # define X509_http_nbio(rctx, pcert) \
-    OSSL_HTTP_REQ_CTX_sendreq_d2i(rctx, (ASN1_VALUE **)(pcert), \
-                                  ASN1_ITEM_rptr(X509))
+    ((*(pcrl) =                                                         \
```

Comment:
Block copy bug here? should this be pcert?

Reply:
:cry:

Reply:
This also implies that these macros are being used exactly nowhere :)

Reply:
Fixed.

> This also implies that these macros are being used exactly nowhere :)

*by us*
It's still a public API.

The thing that brought this up was that I played a bit with a CMakeLists.txt in the 1.1.1 apps, to build against 3.0.
(I have that as a personal learning project to learn cmake)